test/test_execution_engine.cc

```diff
@@ -45,6 +45,24 @@ struct ATenCompilationUnitTest : public ::testing::Test {
   }
 };
 
+TEST_F(ATenCompilationUnitTest, DISABLED_Concat) {
```

Comment:
This is dead code, I see this has been activated in master, still in the future for such cases, please make a test that does something and:
1.  expect the throw / failure that is expected
2. open a proper issue
3. tag the issue in the test as TODO(#number)

Thanks!